ui/scss/component/_video.scss

```diff
@@ -28,6 +27,69 @@ video {
   &.video--active {
     /*background: none;*/
   }
+
+  .video--loading--screen {
+    height: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    .loading--screen--content {
```

Comment:
Since BEM generates unique class names, we don't typically nest CSS styles, particularly classes.